src/components/input.js

```diff
@@ -194,16 +194,25 @@ export const NumberInput = forwardRefWithName('NumberInput', ({ onChange, onBlur
  * @param {object} [props.error] - error message content
  */
 export const ValidatedInput = ({ inputProps, width, error }) => {
+  return createValidatedInput({ inputProps, width, error }, null)
+}
+
+export const ValidatedInputWithRef = forwardRefWithName('ValidatedInput', ({ inputProps, width, error }, ref) => {
+  return createValidatedInput({ inputProps, width, error }, ref)
+})
+
+const createValidatedInput = ({ inputProps, width, error }, ref) => {
```

Comment:
Created this so that I could put keyboard focus in the subscription ID when the wizard is first rendered. Otherwise the user has to tab through the possibly large number of existing billing projects.